frontend/packages/dev-console/integration-tests/features/web-terminal/web-teminal-basic.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Web Terminal
+    As a user, I should be able to use web terminal
+
+
+Background:
+    Given user is at developer perspecitve
+    And user has installed Web Terminal operator
+    And user has created Web Terminal instance
+    And user has created namespace "aut-terminal-basic"
+    And user is at the Topolgy page
+
+
+@regression, @smoke
+Scenario: Web Terminal window
+    Given user is at topology page
+    When user clicks on the Web Terminal icon on the Masthead
+    Then user will see the terminal window
+
+
+@regression
+Scenario: Web Terminal in new tab
+    Given user is at topology page
+    When user clicks on the Web Terminal icon on the Masthead
+    And user clicks on Open Terminal in new tab button
```

Comment:
should `Open Terminal in new tab` be quoted?

Reply:
In this statement, we will be clicking on a button present on the page, so I don't think we should quote anything.